dashboard/app/models/pd/regional_partner_mapping.rb

```diff
@@ -17,6 +18,7 @@
 require 'state_abbr'
 
 class Pd::RegionalPartnerMapping < ActiveRecord::Base
+  acts_as_paranoid # use deleted_at column instead of deleting rows
```

Comment:
I suggest implementing this small change in a separate Pull Request.   We have a general practice of implementing database migrations as standalone changes.  Any application logic that depends on the schema change is implemented as a follow on Pull Request.

Reply:
Here's a similar comment from our former data engineer (although it wasn't ultimately followed 😄:

https://github.com/code-dot-org/code-dot-org/pull/10405#issuecomment-243954041

Reply:
makes sense, will update :)